File: public/usage-examples/json/json_set_number_float/json_set_number_float-1-temperature.py
from splashkit import *

# Create a JSON object
json_obj = create_json()

# Set a float value
json_set_number_float(json_obj, "temperature", 36.6)

# Display the JSON object
write_line("JSON: " + json_to_string(json_obj))

# Free the JSON object
free_json(json_obj)

File: public/usage-examples/json/json_set_number_float/json_set_number_float-1-temperature.cpp
#include "splashkit.h"

int main()
{
    // Create a JSON object
    json json_obj = create_json();

    // Set a float value
    json_set_number(json_obj, "temperature", 36.6f);

    // Display the JSON object
    write_line("JSON: " + json_to_string(json_obj));

    // Free the JSON object
    free_json(json_obj);

    return 0;
}
